### DISCUSSION TITLE
如何阅读由双链笔记组成的文章

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#25] [主发布站的评论系统]
# 如何阅读由双链笔记组成的文章

[!TIP] 阅读小贴士
 目录类的页面，[[MOC]]索引类型的页面，推荐常驻以便随时回到此处：  若是在浏览器中浏览本花园，可以cmd + shift + 点击内部链接，从而新建一个tab页进行阅读和浏览。若你将窗口最大化，则不需要进行上述操作，因为此时会自动启用[[横向卷动布局- andy mode]]，能完整保留阅读上下文。 若是在obsidian中打开本库，可以cmd + p + 输入『锁定』将本页锁定，从而点击内部链接的时候，此页将一直在，使得自己不会迷路。   普通卡片笔记页面：  尽量通过鼠标悬浮预览进行[[上下文]]不中断的阅读。 尽量通过底部[[反向链接]]找回来时的路。     以上的小tips，希望能让你在花园中漫步时，不至于跟随大量链接跳转很快陷入不知所措。 此外，这是我的个人笔记库，而非只关注某个领域的成体系的文章，再加上左侧乱糟糟的文件夹结构，以及页面内大量的跳转链接，你看着感觉乱，实属正常，还请见谅😅。 因此更强烈不推荐按照左侧文件夹的方式去看这些笔记，它不是目录，它不是目录，它不是目录。

https://garden.oldwinter.top/%E5%A6%82%E4%BD%95%E9%98%85%E8%AF%BB%E7%94%B1%E5%8F%8C%E9%93%BE%E7%AC%94%E8%AE%B0%E7%BB%84%E6%88%90%E7%9A%84%E6%96%87%E7%AB%A0

**Comment by keenwon** [comment]
现在怎么没有卷动布局了？？

**Comment by oldwinter** [comment]
唉，官方的发布才有滑动布局，我买了6个月到期了，没续费了。可以期待0.16版本的obsidian，2星期内发布了，自带横向滑动面板哦。

在 2022年9月16日星期五，Keenwon ***@***.***> 写道：
